spreadsheet/formula/fndatetime.go

```diff
@@ -783,14 +783,18 @@ func YearFrac(ctx Context, ev Evaluator, args []Result) Result {
 	if err != nil {
 		return MakeErrorResult("incorrect end date")
 	}
-	return yearFracFromTime(startDate, endDate, basis)
+	yf, errResult := yearFracFromTime(startDate, endDate, basis)
+	if errResult.Type == ResultTypeError {
+		return errResult
+	}
+	return MakeNumberResult(yf)
 }
 
-func yearFrac(startDate, endDate float64, basis int) Result {
+func yearFrac(startDate, endDate float64, basis int) (float64, Result) {
 	return yearFracFromTime(dateFromDays(startDate), dateFromDays(endDate), basis)
 }
 
-func yearFracFromTime(startDate, endDate time.Time, basis int) Result {
+func yearFracFromTime(startDate, endDate time.Time, basis int) (float64, Result) {
```

Comment:
Add a comment to indicate what the return values stand for, what is the float64?

Reply:
fixed